.github/workflows/cli_commands.yml

```diff
@@ -15,236 +15,344 @@ jobs:
               run: dfx start --clean --background --artificial-delay 0
 
             - name: Test stable project creation with azle new command
-              uses: ./azle/.github/actions/test_azle_new
+              uses: ./.github/actions/test_azle_new
               with:
-                  project_name: 'stable_project_test'
+                  project-name: 'stable_project_test'
                   experimental: 'false'
-                  http_server: 'false'
-                  working_directory: ${{ steps.create_test_environment.outputs.external_test_directory }}
-                  version: ${{ env.UNIQUE_VERSION }}
-                  packed_file_path: ${{ steps.pack_azle_source.outputs.packed_file_absolute_path }}
+                  http-server: 'false'
+                  working-directory: ${{ steps.create-external-test-env.outputs.external-test-env-dir-path }}
+                  version: ${{ steps.create-test-version.outputs.test-version }}
+                  packed-file-path: ${{ steps.pack-azle-source.outputs.packed-file-absolute-path }}
 
             - name: Test experimental project creation with azle new command
-              uses: ./azle/.github/actions/test_azle_new
+              uses: ./.github/actions/test_azle_new
               with:
-                  project_name: 'experimental_project_test'
+                  project-name: 'experimental_project_test'
                   experimental: 'true'
-                  http_server: 'false'
-                  working_directory: ${{ steps.create_test_environment.outputs.external_test_directory }}
-                  version: ${{ env.UNIQUE_VERSION }}
-                  packed_file_path: ${{ steps.pack_azle_source.outputs.packed_file_absolute_path }}
+                  http-server: 'false'
+                  working-directory: ${{ steps.create-external-test-env.outputs.external-test-env-dir-path }}
+                  version: ${{ steps.create-test-version.outputs.test-version }}
+                  packed-file-path: ${{ steps.pack-azle-source.outputs.packed-file-absolute-path }}
 
             - name: Test HTTP server project creation with azle new command
-              uses: ./azle/.github/actions/test_azle_new
+              uses: ./.github/actions/test_azle_new
               with:
-                  project_name: 'http_server_project_test'
+                  project-name: 'http_server_project_test'
                   experimental: 'true'
-                  http_server: 'true'
-                  working_directory: ${{ steps.create_test_environment.outputs.external_test_directory }}
-                  version: ${{ env.UNIQUE_VERSION }}
-                  packed_file_path: ${{ steps.pack_azle_source.outputs.packed_file_absolute_path }}
+                  http-server: 'true'
+                  working-directory: ${{ steps.create-external-test-env.outputs.external-test-env-dir-path }}
+                  version: ${{ steps.create-test-version.outputs.test-version }}
```

Comment:
Change the name of this input to `expected-version`

Reply:
done